delphi-frontend/src/main/java/au/com/integradev/delphi/cfg/api/ExitPath.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Sonar Delphi Plugin
+ * Copyright (C) 2024 Integrated Application Development
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
+ */
+package au.com.integradev.delphi.cfg.api;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public interface ExitPath extends Successors {
+  /** Next block without exceptional circumstances */
```

Comment:
Across the board, I think we need to tidy the javadocs in the API layer.
- aim for less terse wording
- use `@param` and `@return` tags
- put javadocs on the types themselves and not just the methods